dependabot/util.py

```diff
@@ -0,0 +1,16 @@
+import github3.repos
+import typing
+
+import ccc.github
+
+
+def repositories_for_org(
+    full_org_name: str,  # e.g. github.com/gardener
+) -> typing.List[github3.repos.repo.ShortRepository]:
+
+    host, org = full_org_name.split('/')
```

Comment:
rather conditionally prepend a schema (if absent) + use urllib.parse.urlparse to parse into host and path?